requirements.txt

```diff
@@ -0,0 +1,7 @@
+typing
```

Comment:
`pyproject.toml` already exists and specifies the dependency for [poetry](https://python-poetry.org/). This file is not necessary.